README.rst

```diff
@@ -49,6 +49,39 @@ To scaffold an application called ``myapp`` in the current directory do::
 
    $ molo scaffold myapp .
 
+Specifying extra requires
+=========================
+
+Molo in itself is not expected to be enough to deliver on a client request.
+During scaffolding use the ``--require`` commandline parameter to include
+more libraries that are required for installation::
+
+   $ molo scaffold myapp --require=django-contrib-comments
```

Comment:
@smn This is a bit of a problem because after `pip install django-contrib-comments` you add `django_comments` to the `INSTALLED_APPS` so we almost need a way to specify the module name e.g `--require=django-contrib-comments#django_comments`?